### DISCUSSION TITLE
Rule for play mp3 track

### DISCUSSION THREAD

**Discussion by lisergio** [arendst/Tasmota#12228] [Q&A] [answered]
Hello
I am trying to create a rule, so that a track in mp3 is played when the output is activated ..

If I write in the console "MP3Track 1" the track plays correctly ...

I have created the following Rule:
Rule1 on power # state = 1 do MP3Track 1
Rule1 1

but it does not reproduce ...

any ideas

Thank you

**Comment by ascillato** [comment, marked as answer]
Hi,

You have a syntax error in your rule. It should be:

```lua
Rule1 1
Rule1 on power1#state=1 do MP3Track 1 endon
```

**Reply by lisergio** [reply]
It's true!
thank you very much ...
